static/src/javascripts/boot.js

```diff
@@ -75,5 +75,21 @@ domready(() => {
             },
             'commercial'
         );
+    } else if (window.guardian.isEnhanced) {
```

Comment:
ooooh dammit, I didn't you did not get rid of the `then` part above. All is fine then.